boot_node/src/config.rs

```diff
@@ -76,7 +76,7 @@ impl<T: EthSpec> TryFrom<&ArgMatches<'_>> for BootNodeConfig<T> {
             let spec = eth2_network_config
                 .yaml_config
                 .as_ref()
-                .ok_or_else(|| "The network directory must contain a spec config".to_string())?
+                .ok_or("The testnet directory must contain a spec config")?
```

Comment:
We've renamed this to `network` right? The `--network` command. I see it does conflict with the actual networking directory, but the text here is correct right?

Reply:
you are right, done